content/en/real_user_monitoring/browser/_index.md

```diff
@@ -67,6 +67,8 @@ datadogRum.init({
   trackResources: true,
   trackLongTasks: true,
   trackUserInteractions: true,
+  });
+DD_RUM.startSessionReplayRecording();
 })
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** For CDN snippets, syntax issue with extra curly bracket and parenthesis + inappropriate global variable, it should be
```js
datadogRum.init({
 ...
});
datadogRum.startSessionReplayRecording();
```
‏